pudl/extract/epaipm.py

```diff
@@ -26,8 +27,16 @@ def get_epaipm_name(file):
     Returns:
         path to EPA IPM spreadsheet.
     """
+    if sorted(datadir.glob(file)):
+        name = sorted(datadir.glob(file))[0]
+    elif sorted(pudl_datadir.glob(file)):
+        name = name = sorted(pudl_datadir.glob(file))[0]
```

Comment:
Is this double assignment of `name` intentional?

Reply:
Nope. Thanks for the catch.